tests/installation/bootloader_s390.pm

```diff
@@ -86,7 +86,7 @@ sub prepare_parmfile {
 
     # we have to hardcode the hostname here - the true hostname would
     # create a too long parameter ;(
-    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') . '://' . get_var('REPO_HOST', 'openqa') . '/';
+    my $instsrc = get_var('INSTALL_SOURCE', 'ftp') . '://ftp:ftp@' . get_var('REPO_HOST', 'openqa') . '/';
```

Comment:
The only test suite I have found that uses **INSTALL_SOURCE** is for x86_64: https://openqa.suse.de/tests/3355028#step/bootloader/10

So, I suppose it will not brake anything.

Reply:
uhm... but what if tomorrow someone will introduce a test with INSTALL_SOURCE=HTTP ?
he will end up with connecting to "http://ftp:ftp@...."

Reply:
I was confused because this is working magically without user and password: https://openqa.suse.de/tests/3354202#step/bootloader_s390/17

And I thought it was always set to http

Reply:
> I was confused because this is working magically without user and password: https://openqa.suse.de/tests/3354202#step/bootloader_s390/17
> 
> And I thought it was always set to http

yup - so actually we wouldn't need this change at all - because the ftp server allows anonymous.

However - we can still implement it explicitly for the ftp case to be on the safe side, once the ftp-server settings are changed

Reply:
But, there has to be a special FTP setup there.
I was/am having the same problem as Mike when trying the FTP installation media. All FTP servers I tried (local setup), force you to specify the anonymous user and password, in this case, ftp:ftp.

Reply:
Thank you for catching my stupid mistake!